app/src/main/java/com/github/se/studybuddies/viewModels/ToDoListViewModel.kt

```diff
@@ -33,9 +33,7 @@ class ToDoListViewModel(studyBuddies: Application) : AndroidViewModel(studyBuddi
   private val json = Json { prettyPrint = true }
   // private val gson = Gson()
 
-  private val toDoFile = File(studyBuddies.filesDir, "ToDoList.json")
-
-  // val encryption = Encryption(studyBuddies)
+  private val toDoFile = File(context.filesDir, "ToDoList.json")
 
   init {

```

Comment:
Just in case, the coaches sent a very useful link about viewmodels I think two weeks ago and it said that using init is bad practice for viewmodels as they could cause problems in the tests (though it seems to be working fine here) so just a head's up !

Reply:
Okay thank you, I will check that, might have time to update it if useful